10-validation.md

```diff
@@ -20,7 +20,7 @@ This section will help you to validate the workload is exposed correctly and res
 1. Browse to the site.
 
    ```bash
-   echo $FRONTDOOR_FQDN
+   echo https://$FRONTDOOR_FQDN
```

Comment:
please let's revert this change to avoid conflicts. I'm already taking care of this section from another PR.

Reply:
done